spec/factories/user.rb

```diff
@@ -0,0 +1,17 @@
+FactoryBot.define do
+  factory :user, class: User do
+    first_name              { 'john' }
+    last_name               { 'doe' }
+    email                   { 'test@example.com' }
+    password                { '123456' }
```

Comment:
you can use Faker for password and repeat it in confirmation doing
`password_confirmation { password }`